File: NodeJS Testing/checking_CPU_temp.js
const { exec } = require('child_process');

function getData()
{
	exec('date',(err,out)=>{if(err){console.error(err)}else{console.log(out)}});
//	exec('vcgencmd measure_temp',(err,out)=>{if(err){console.error(err)}else{console.log(out)}});
	console.log('');
}

console.log('Getting data from CPU ...');
setInterval(getData,1000);

File: NodeJS Testing/checking_CPU_temp.py
import os
import time

def measure_temp():
        temp = os.popen("vcgencmd measure_temp").readline()
        return (temp.replace("CPU temperature: ",""))

def get_time():
        dtime = os.popen("date").readline()
        return (dtime.replace("Date time: ",""))

while True:
        print(get_time())
        print(measure_temp())
        time.sleep(1)


